plugin.go

```diff
@@ -44,6 +44,9 @@ func (p Plugin) Exec() error {
 
 	var commands []*exec.Cmd
 	remote := p.Config.Remote
+	if len(p.Config.Secrets) != 0 {
+		exportSecrets(p.Config.Secrets)
```

Comment:
My logic is this:

I have one terraform repo that can be run against 5 or so environments. Each environment has a different set of secrets to be able to apply to them. I can add org level secrets that are for example

DEV_SECRET_KEY
DEV_ACCESS_KEY
NPE_SECRET_KEY
NPE_ACCESS_KEY
PROD_SECRET_KEY
PROD_ACCESS_KEY

I can't put them all in as AWS_ACCESS_KEY_ID or AWS_SECRET_ACCESS_KEY for the same repo as you'd have to change them to run against each environment every time.

This way you can set those via differently named secrets as AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY and assign the right credential in the secrets section.

Does this make sense, or is there a work around I was not aware of?